bdtopo_v2/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/bdtopo_v2/InputDataFormattingTest.groovy

```diff
@@ -134,145 +133,136 @@ class InputDataFormattingTest {
         // -------------------------------
-        // ... with the building 'BATIMENT0000000290114260' : HAUTEUR = 0 / TYPE = 'Bâtiment sportif'
+        // ... with the building 'BREMAR0001' : HAUTEUR = 0 / TYPE = 'Bâtiment sportif'
         assertEquals(3, h2GISDatabase.firstRow("SELECT HEIGHT_WALL FROM BUILDING " +
-                "WHERE ID_SOURCE='BATIMENT0000000290114260';")["HEIGHT_WALL"])
+                "WHERE ID_SOURCE='BREMAR0001';")["HEIGHT_WALL"])
         assertEquals(3, h2GISDatabase.firstRow("SELECT HEIGHT_ROOF FROM BUILDING " +
-                "WHERE ID_SOURCE='BATIMENT0000000290114260';")["HEIGHT_ROOF"])
+                "WHERE ID_SOURCE='BREMAR0001';")["HEIGHT_ROOF"])
         assertEquals(1, h2GISDatabase.firstRow("SELECT NB_LEV FROM BUILDING " +
-                "WHERE ID_SOURCE='BATIMENT0000000290114260';")["NB_LEV"])
+                "WHERE ID_SOURCE='BREMAR0001';")["NB_LEV"])
         assertEquals('sports_centre', h2GISDatabase.firstRow("SELECT TYPE FROM BUILDING " +
-                "WHERE ID_SOURCE='BATIMENT0000000290114260';")["TYPE"])
+                "WHERE ID_SOURCE='BREMAR0001';")["TYPE"])
 
-        //... with the building 'BATIMENT0000000290116893' : HAUTEUR = 0 / NATURE = 'Tour, donjon, moulin'
+        //... with the building 'BREMAR0002' : HAUTEUR = 0 / NATURE = 'Tour, donjon, moulin'
         assertEquals(3, h2GISDatabase.firstRow("SELECT HEIGHT_WALL FROM BUILDING " +
-                "WHERE ID_SOURCE='BATIMENT0000000290116893';")["HEIGHT_WALL"])
+                "WHERE ID_SOURCE='BREMAR0002';")["HEIGHT_WALL"])
         assertEquals(3, h2GISDatabase.firstRow("SELECT HEIGHT_ROOF FROM BUILDING " +
-                "WHERE ID_SOURCE='BATIMENT0000000290116893';")["HEIGHT_ROOF"])
+                "WHERE ID_SOURCE='BREMAR0002';")["HEIGHT_ROOF"])
         assertEquals(1, h2GISDatabase.firstRow("SELECT NB_LEV FROM BUILDING " +
-                "WHERE ID_SOURCE='BATIMENT0000000290116893';")["NB_LEV"])
+                "WHERE ID_SOURCE='BREMAR0002';")["NB_LEV"])
         assertEquals('historic', h2GISDatabase.firstRow("SELECT TYPE FROM BUILDING " +
-                "WHERE ID_SOURCE='BATIMENT0000000290116893';")["TYPE"])
+                "WHERE ID_SOURCE='BREMAR0002';")["TYPE"])
 
-        //... with the building 'BATIMENT0000000290116890' : HAUTEUR = 12 / NATURE = 'Chapelle'
+        //... with the building 'BREMAR0003' : HAUTEUR = 12 / NATURE = 'Chapelle'
         assertEquals(12, h2GISDatabase.firstRow("SELECT HEIGHT_WALL FROM BUILDING " +
-                "WHERE ID_SOURCE='BATIMENT0000000290116890';")["HEIGHT_WALL"])
+                "WHERE ID_SOURCE='BREMAR0003';")["HEIGHT_WALL"])
         assertEquals(12, h2GISDatabase.firstRow("SELECT HEIGHT_ROOF FROM BUILDING " +
-                "WHERE ID_SOURCE='BATIMENT0000000290116890';")["HEIGHT_ROOF"])
+                "WHERE ID_SOURCE='BREMAR0003';")["HEIGHT_ROOF"])
         assertEquals(1, h2GISDatabase.firstRow("SELECT NB_LEV FROM BUILDING " +
```

Comment:
You can simplify the test  to return multiple value for the same id

def values = h2GISDatabase.firstRow "SELECT HEIGHT_ROOF, GWEN_GROOVE, JEJE_NOT_WORKING FROM BUILDING    WHERE ID_SOURCE='BATIMENT0000000290116890'"

and do assert on 
values.HEIGHT_ROOF
values.GWEN_GROOVE
values.JEJE_NOT_WORKING

Reply:
Okay, I didn't know that. I'm gonna look into it

Reply:
Done, with the help from @SPalominos